js/apps/system/_admin/aardvark/APP/frontend/js/views/indicesView.js

```diff
@@ -9,15 +9,39 @@
     el: '#content',
 
     initialize: function (options) {
+      var self = this;
       this.collectionName = options.collectionName;
       this.model = this.collection;
+
+      // rerender
+      self.interval = window.setInterval(function () {
+        if (window.location.hash.indexOf('cIndices/' + self.collectionName) !== -1 && window.VISIBLE) {
+          if ($('#collectionEditIndexTable').is(':visible') && !$('#indexDeleteModal').is(':visible')) {
+            self.rerender();
+          }
+        }
+      }, self.refreshRate);
     },
 
+    interval: null,
+    refreshRate: 3000,
```

Comment:
Do we really want to send requests to the database every 3 seconds?
I think we should avoid that. If we definitely want to repeatedly check for new indexes, we should change the frequency to something like at least 10 seconds.

Reply:
Fine for me.